### PR TITLE
[dependabot][mergify] automate merge and add reviewers

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

- Add reviewers
- Automerge if CI is happy. Although, if no approval is needed why do we need the reviewers?

## Why is it important?

Look mum no hands!

## Issue

See https://github.com/elastic/integrations/pull/1291#issuecomment-876963997

### CODE REVIEW

**Review by cachedout — APPROVED** [review]

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,6 +6,8 @@ updates:
     schedule:
       interval: "daily"
     labels:
-      - dependencies
-      - go
+      - automation
+    reviewers:
+      - "mtojek"
+      - "jsoriano"
```
Comment (jsoriano) [inline .github/dependabot.yml:12]: I'd prefer to put a [github] team here so we can more easily manage notifications and we don't need need to change this if the people related to the project changes. But we can go on like this by now.

**Review by kuisathaverat — APPROVED** [review]

**Review by mtojek — APPROVED** [review]

File: .github/dependabot.yml (hunk 1 above)
Comment (jsoriano) [reply .github/dependabot.yml:12]: @v1v and I have discussed about this today and we have created a new "integrations-developer-experience" github subteam for this, so we can use it to assign reviewers.

@mtojek sounds this approach good to you?

Comment (mtojek) [reply .github/dependabot.yml:12]: SGTM! I like the idea :)

Comment (v1v) [reply .github/dependabot.yml:12]:
```suggestion
      - "elastic/integrations-developer-experience"
```

**Review by mtojek — APPROVED** [review]

**Review by jsoriano — APPROVED** [review]

**Pull request merged**